lib/helpers.js

```diff
@@ -188,6 +188,39 @@ export function deepEqual(obj1, obj2) {
   return true;
 }
 
+export function createVerifyOptions({
+  verifyPresentationOptions,
+  expectedChallenge,
+  verifiablePresentationRequest,
+  presentation,
+  domain,
+  checks
+}) {
+  // start with verifyPresentationOptions, then overwrite as needed
+  const options = {...verifyPresentationOptions};
+  // update 'checks' with anything additional from 'verifyPresentationOptions'
+  if(verifyPresentationOptions.checks !== undefined) {
+    for(const check in verifyPresentationOptions.checks) {
+      if(verifyPresentationOptions.checks[check] == true) {
+        if(!checks.includes(check)) {
+          checks.push(check);
+        }
+      }
+    }
+  }
+  options.checks = checks;
```

Comment:
I think I like Tyler's suggestion better (seems more readable), but if you were looking for a functional way to do it, it could look like this:

```suggestion
  options.checks = checks;
  // update 'checks' with anything additional from 'verifyPresentationOptions'
  if(verifyPresentationOptions.checks) {
    options.checks = [...new Set([
      ...options.checks, 
      ...Object.entries(verifyPresentationOptions.checks)
        .filter(([, v]) => v).map(([k]) => k)]
    ])];
  }
```